components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.ts

```diff
@@ -99,7 +99,7 @@ export class ProjectsFilterDropdownComponent {
     this.checkInitialEquality();
   }
 
-  handleArrowUp(event: KeyboardEvent) {
+  handleArrowUp(event: Event) {
```

Comment:
@msorens - Would you happen to have any advice on this?  Typescript is complaining about $event coming in as type `KeyboardEvent`.  Event "fixes" the problem but it feels way too broad since this is specifically a keyboard event.  
error thrown reads as:
```
error TS2345: Argument of type 'Event' is not assignable to parameter of type 'KeyboardEvent'.
```

Reply:
Apologies; did not see this before. Where are you seeing the Typescript error?

Reply:
Thanks for the details. Here is my suggestion:

Because we cannot cast in a template, we have to accept the argument as a generic event. So let's use that and then redirect it to what we need.
```suggestion
  handleArrowUp(genericEvent: Event) {
    const event = genericEvent as KeyboardEvent;
 ```

Additional notes:
1. For the specific functions `handleArrowUp` (and `handleArrowDown`) it is actually OK to use the signature `handleArrowUp(event: Event)` because the only properties we are using in the body of the function are, in fact, properties of the base class `Event`. That said, there is no harm in making the above change as one could argue it does signal intent a bit more clearly. Your call.

2. There are a couple occurrences of the same error though in another file (resource-dropdown.component.ts) you should remedy in this PR. The original code is this:
```
  moveFocus(event: KeyboardEvent): void {
```
And that does need the change above:
```
  moveFocus(genericEvent: Event): void {
    const event = genericEvent as KeyboardEvent;
```
because it uses properties that are specific to the `KeyboardEvent` subclass.

Reply:
Thank you for this information, It makes a lot of sense and seems like a good way to move forward.  Much appreciated!